src/templates/specs/engineering.md

```diff
@@ -1,15 +1,10 @@
 # Engineering Principles for {project-name}
 
 > [INSTRUCTIONS]
-> This document defines engineering principles and standards for {project-name}. Keep the list to core principles. Focus on timeless, actionable guidelines that guide AI and human development with an emphasis on quality. reliability, performance, security, etc. Remove all instruction blocks after completion.
->
-> **IMPORTANT - Minimize Token Usage**: The principle definitions below are sufficient for most projects. Only add project-specific context (1 sentence max) if the principle has a unique interpretation or constraint for {project-name}. Otherwise, keep principles as-is without additional rationale.
+> Define engineering principles and standards. Focus on timeless, actionable guidelines that guide AI and human development. Remove any principles that don't fit {project-name}. Minimize token usage by keeping principles concise.
```

Comment:
Remove placeholders from instructions. They don't need to be replaced. Use "this project" or similar.